packages/gatsby-theme-newrelic/src/components/GlobalHeader.js

```diff
@@ -27,7 +30,20 @@ const styles = {
 };
 
 const GlobalHeader = ({ editUrl, className, search }) => {
-  const [isOverlayOpen, setIsOverlayOpen] = useState(false);
+  const [isOverlayOpen, setIsOverlayOpen] = useState(null);
+
+  const [isQueryChanged, setIsQueryChanged] = useState(false);
+
+  useEffect(() => {
+    let searchQuery = qs.parse(window.location.search);
```

Comment:
[nit] If you don't want to pull in an npm package, you can also use [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) which is built into the browser as a way to parse a query string. Totally fine if you want to use `qs` since it has a nice API, but just wanted you to be aware the other one exists as well :)